docs/framework-introscope_agent.md

```diff
@@ -23,9 +23,9 @@ The credential payload of the service may contain the following entries:
 
 | Name | Description
 | ---- | -----------
-| `agent-name` | (Optional) The name that should be given to this instance of the Introscope agent
 | `url` | The url of the Introscope Enterprise Manager server
-
+| `credential`| (Optional) The credential that is used to connect to the Enterprise Manager server
+| `agent_name` | (Optional) The name that should be given to this instance of the Introscope agent
```

Comment:
This whole list should be alphabetical.  At the very least, no less alphabetical than it was 😄

Reply:
Oh I see, it was originally alphabetical, but I re-ordered it according to the optionality of the property. I will make it alphabetical again.

Reply:
Yep, we stick to lexical sorting for predictability.